### PR TITLE
chore: add CONTRIBUTING and CODE_OF_CONDUCT files

### LINKED ISSUE #22: Add CONTRIBUTING file
A CONTRIBUTING file is needed in order to tell contributors how to move inside the repo.

### PR DESCRIPTION
**Pull request by libremente**
This PR:
* adds a CONTRIBUTING file
* adds a CODE_OF_CONDUCT file

Fixes #22

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+If a commit fixes an issue, please
+[reference it](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords#about-issue-references)
+ in the commit body with `Fix: #ISSUENUMBER`.
+
+## Gitflow
+
+This repository adopts a simplified branch management system as follows:
+
+* `master` is stable and gets deployed automatically. Never push directly to it;
```
Comment (bfabio) [inline CONTRIBUTING.md:72]:
```suggestion
* `main` is stable and gets deployed automatically. Never push directly to it;
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+> :information_source: Please ensure that there is a pertinent issue related to
+> what you are proposing and also make sure that someone has already reviewed it
+> before proceeding
+
+1. [Fork the project](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
+
+2. Follow our [conventions regarding commits](#commit-messages) for your commit message
+
+3. Open a Pull request against `master`.
```
Comment (bfabio) [inline CONTRIBUTING.md:41]:
```suggestion
3. Open a Pull request against `main`.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+file a new one: we really appreciate it! :rocket:.
+
+You're welcome to contribute to open issues with more information or by adding
+:+1: on them: it will help the maintainers identify the issues to be prioritized.
+
+### Creating a new issue
+
+There are three templates for new issues: `Bug report`, `Feature request`
+and `General issue`, pick the relevant one and follow the instructions
+in that template.
```
Comment (bfabio) [inline CONTRIBUTING.md:21]: We don't have templates on this repo

Comment (libremente) [reply CONTRIBUTING.md:21]: Let's get rid of that part then, we'll insert those templates later on, thanks

**Review by bfabio — APPROVED** [review]

**Pull request merged**